assets/src/containers/DashboardAppBar.js

```diff
@@ -51,7 +51,7 @@ function DashboardAppBar (props) {
 
   return (
     <div>
-      <AppBar className={classes.root} position='static'>
+      <AppBar className={classes.root} position='fixed'>
```

Comment:
This change is chopping off the Title of all the views like I don't see `Resource access` title at all

Reply:
May be try adding a margin as suggested here https://stackoverflow.com/a/43795590/724238